src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -234,13 +250,19 @@ private function tagsAssets(array $ids, string $tag): void
             }
         }
 
-        // Set tags for all ids without comma.
-        $response = $this->client->request('POST', sprintf('tags/%s.json', rawurlencode($tag)), [
-            'body' => implode(',', $idsWithoutComma),
-        ]);
+        if ([] !== $idsWithoutComma) {
```

Comment:
Avoid calling provider when `idsWithoutComma` is empty. There is no asset to tag so I think is ok :blush: 

https://localise.biz/api/docs/tags/tagassets